### ISSUE TITLE
Kdenlive fails to build: Could NOT find KF5Purpose

### ISSUE THREAD

**Issue by deftdawg** [KDE-mac/homebrew-kde#338]
# Please specify exact formula that has issue

kdenlive

System: OSX 10.15.4

# What happened (include command output)

Build command:
```sh
brew install kde-mac/kde/kdenlive
"$(brew --repo kde-mac/kde)/tools/do-caveats.sh"
brew install kde-mac/kde/kdenlive
```

End of build output:
```
....
==> Pouring sox-14.4.2_3.catalina.bottle.tar.gz
🍺  /usr/local/Cellar/sox/14.4.2_3: 23 files, 1.9MB
==> Installing kde-mac/kde/kdenlive dependency: mlt
==> Downloading https://homebrew.bintray.com/bottles/mlt-6.20.0.catalina.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/57/5703d5533277335653085a3beed96f9df7053865b49d069d2d9fc2106550c8f2?__gda__=exp=1587607395~hmac=f5ba077499762c3e4e2a8d4a18c8f20d6d26c839bd99a7a477f027958b
######################################################################## 100.0%
==> Pouring mlt-6.20.0.catalina.bottle.tar.gz
🍺  /usr/local/Cellar/mlt/6.20.0: 400 files, 2.6MB
==> Installing kde-mac/kde/kdenlive
==> Downloading https://download.kde.org/stable/release-service/19.12.3/src/kdenlive-19.12.3.tar.xz
==> Downloading from http://mirror.csclub.uwaterloo.ca/kde/stable/release-service/19.12.3/src/kdenlive-19.12.3.tar.xz
######################################################################## 100.0%
==> Patching
patching file data/CMakeLists.txt
Hunk #1 succeeded at 29 (offset -4 lines).
==> cmake -G Ninja .. -DKDE_INSTALL_BUNDLEDIR=/usr/local/Cellar/kdenlive/19.12.3/bin -DKDE_INSTALL_LIBDIR=lib -DKDE_INSTALL_QMLDIR=lib/qt5/qml -DKDE_INSTALL_PLUGINDIR=lib/qt5/plugins -DBUILD_TESTING=OFF
Last 15 lines from /Users/ged/Library/Logs/Homebrew/kdenlive/01.cmake:
-- Found KF5FileMetaData: /usr/local/lib/cmake/KF5FileMetaData/KF5FileMetaDataConfig.cmake (found version "5.69.0")
-- Found KF5Crash: /usr/local/lib/cmake/KF5Crash/KF5CrashConfig.cmake (found version "5.69.0")
-- Could NOT find KF5Purpose (missing: KF5Purpose_DIR)
-- Could NOT find KF5Purpose: found neither KF5PurposeConfig.cmake nor kf5purpose-config.cmake
CMake Error at /usr/local/Cellar/cmake/3.17.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:164 (message):
  Could NOT find KF5 (missing: Declarative) (found version "5.69.0")
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.17.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:445 (_FPHSA_FAILURE_MESSAGE)
  /usr/local/share/ECM/find-modules/FindKF5.cmake:110 (find_package_handle_standard_args)
  CMakeLists.txt:54 (find_package)


-- Configuring incomplete, errors occurred!
See also "/tmp/kdenlive-20200422-19773-1xgrlq9/kdenlive-19.12.3/build/CMakeFiles/CMakeOutput.log".
See also "/tmp/kdenlive-20200422-19773-1xgrlq9/kdenlive-19.12.3/build/CMakeFiles/CMakeError.log".

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/kde-mac/homebrew-kde/issues
```

the output and error logs are deleted on exit before they can be examined or collected to attach.

**Comment by yurikoles** [comment]
fixed in c6b6ea1fd6894361158f9936e40cc33b3d6f9d4c

**Comment by yurikoles** [comment]
@deftdawg, thanks for report!
